fen.go

```diff
@@ -26,8 +26,8 @@ func (w *Watcher) Close() error {
 	return nil
 }
 
-// Add starts watching the named file or directory (non-recursively).
-func (w *Watcher) Add(name string) error {
+// AddRaw starts watching the named file or directory (non-recursively). Symlinks are not implicitly resolved.
+func (w *Watcher) AddRaw(name string) error {
 	return nil
```

Comment:
this should probably do something

Reply:
probably - I suppose it is covered by this existing [issue](https://github.com/fsnotify/fsnotify/issues/12)